hll/include/HllSketchImplFactory.hpp

```diff
@@ -157,8 +159,9 @@ Hll6Array<A>* HllSketchImplFactory<A>::convertToHll6(const HllArray<A>& srcHllAr
 template<typename A>
 Hll8Array<A>* HllSketchImplFactory<A>::convertToHll8(const HllArray<A>& srcHllArr) {
   const int lgConfigK = srcHllArr.getLgConfigK();
-  typedef typename std::allocator_traits<A>::template rebind_alloc<Hll8Array<A>> hll8Alloc;
-  Hll8Array<A>* hll8Array = new (hll8Alloc().allocate(1)) Hll8Array<A>(lgConfigK, srcHllArr.isStartFullSize());
+  using Hll8Alloc = typename std::allocator_traits<A>::template rebind_alloc<Hll8Array<A>>;
+  Hll8Array<A>* hll8Array = new (Hll8Alloc(srcHllArr.getAllocator()).allocate(1))
```

Comment:
I'm always torn on whether we should define `auto srcAllocator = srcHllArr.getAllocator()` in cases like this since we use it twice and it gets a little verbose to read. Two uses like this is sort of a wash I guess?

It's perfectly fine as is, I'm just not sure if we should have a preferred style?

Reply:
I prefer not to create aliases without a good reason.